redisdb/tests/test_unit.py

```diff
@@ -73,3 +73,33 @@ def test__check_command_stats_host(
         'redis.command.usec_per_call', value=expected_usec_per_call_value, count=1, tags=expected_tags
     )
     aggregator.assert_metrics_using_metadata(get_metadata_metrics())
+
+
+def test__check_total_commands_processed_not_present(check, aggregator, redis_instance):
+    """
+    The check shouldn't send the `redis.net.commands` metric if `total_commands_processed` is not present in `c.info`
+    """
+    redis_check = check(redis_instance)
+    conn = mock.MagicMock()
+    conn.info.return_value = {}
+
+    # Run the check
+    redis_check._check_total_commands_processed(conn.info(), [])
+
+    # Assert that no metrics were sent
+    assert len(list(aggregator.metrics('redis.net.commands'))) == 0
```

Comment:
```suggestion
    aggregator.assert_metric('redis.net.commands', count=0)
```

Nit- I think it would be easier that way 🙂